README.md

```diff
@@ -128,3 +128,20 @@ The workflow returns the following outputs:
 output | description
 --- | ---
 md5_check | String with results of check (PASS, FAIL, or UNVERIFIED)
+
+
+## backup_data_tables
+
+Workflow to back up the data currently stored in data tables in a workspace to a
+folder in the cloud bucket associated with that workspace. The data tables are
+exported in tsv format. A json file is also provided containing the mapping between
+data table name and output file, which could be used as an input to validate_data_model
+if the user wishes to restore data tables using these tsvs.
+
+The user must specify the following inputs:
+
+input | description
+--- | ---
+workspace_name | A string with the workspace name. e.g, if the workspace URL is https://anvil.terra.bio/#workspaces/fc-product-demo/Terra-Workflows-Quickstart, the workspace name is "Terra-Workflows-Quickstart"
+workspace_namespace | A string with the workspace name. e.g, if the workspace URL is https://anvil.terra.bio/#workspaces/fc-product-demo/Terra-Workflows-Quickstart, the workspace namespace is "fc-product-demo"
+output_directory | The name of the folder in which data table tsvs will be stored. If the folder does not to exist in the workspace bucket, the workflow will create it.
```

Comment:
Note: add description of outputs if we decide to have the workflow produce outputs.

Reply:
typo: "does not to exist"